WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -472,24 +457,10 @@ private extension StoreStatsV4PeriodViewController {
                 reloadSiteVisitUI()
                 return
             }
-            guard visitorsData != nil else {
+            guard visitorsDataOrRedactedView != nil else {
```

Comment:
I think these checks on view nullability is because the period view controllers (`StoreStatsV4PeriodViewController`s) are initialized without being shown and some public functions/variables might trigger UI updates. The view controllers are also managed by `XLPagerTabStrip.ButtonBarPagerTabStripViewController`, maybe we keep these checks for now to prevent from crashes 😅